### PR TITLE
Update AuthorNotification.md

### PR DESCRIPTION
**Pull request by v-caxian**

### CODE REVIEW

**Comment by PRMerger20** [comment]
@v-caxian : Thanks for your contribution! The author, @PRMerger16, has been notified to review your proposed change.

File: CSIDevDocsPublic/AuthorNotification.md [hunk 1]
```diff
@@ -42,6 +42,6 @@ VS Code Markdown Authoring Extension for OPS is a poc to provide authoring help
 ## Prerequisites and assumptions
 
 To effectively use the OPS Authoring extension, you must:
-- Clone your entire repo to your local machine and keep it in sync. Functions such as link and image insertion are not reliable if the repo is out of sync.
+Clone your entire repo to your local machine and keep it in sync. Functions such as link and image insertion are not reliable if the repo is out of sync.
 - For accurate content preview, you need to update your VS Code settings.json file, as described under **Preview Content** below.
```
Comment (PRMerger16) [inline CSIDevDocsPublic/AuthorNotification.md]:
```suggestion
accurate content preview, you need to update your VS Code settings.json file, as described under **Preview Content** below.
```

**Comment by PRMerger20** [comment]
@PRMerger16 : Thanks for your contribution! The author, @PRMerger16, has been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@PRMerger16 : Thanks for your contribution! The author, @PRMerger16, has been notified to review your proposed change.

**Pull request closed without merging**